src/main/java/com/synopsys/integration/alert/channel/azure/boards/service/AzureCustomFieldManager.java

```diff
@@ -46,13 +49,43 @@
 import com.synopsys.integration.azure.boards.common.service.project.TeamProjectReferenceResponseModel;
 
 public class AzureCustomFieldManager {
-    public static final String ALERT_TOP_LEVEL_KEY_FIELD_NAME = "Alert Top Level Key";
-    public static final String ALERT_TOP_LEVEL_KEY_FIELD_REFERENCE_NAME = "Custom.AlertTopLevelKey";
-    public static final String ALERT_TOP_LEVEL_KEY_FIELD_DESCRIPTION = "A top-level tracking key for Alert";
-
-    public static final String ALERT_COMPONENT_LEVEL_KEY_FIELD_NAME = "Alert Component Level Key";
-    public static final String ALERT_COMPONENT_LEVEL_KEY_FIELD_REFERENCE_NAME = "Custom.AlertComponentLevelKey";
-    public static final String ALERT_COMPONENT_LEVEL_KEY_FIELD_DESCRIPTION = "A component-level tracking key for Alert";
+    public static final String ALERT_PROVIDER_KEY_FIELD_NAME = "Alert Provider Key";
+    public static final String ALERT_PROVIDER_KEY_FIELD_REFERENCE_NAME = "Custom.AlertProviderKey";
```

Comment:
Do these references name need to be as unique as possible? If so, we might want to add synopsys or something to the name as well so that there are no conflicts.

Reply:
Well yes, but actually no. This is a risk we run, but the reference name is just the context plus a dot prefixed to the name of the field without spaces.

Reply:
@jamesrichard91 What are your thoughts? These are user-visible fields, so it will potentially make issues more verbose, but that could be a good thing so that customers aren't confused.